src/Commands/BuildToolsBase.php

```diff
@@ -416,6 +416,8 @@ protected function useExistingSourceDirectory($source, $preserve_local_repositor
     protected function createFromSourceProject($source, $target, $stability = '', $template_repository = '')
     {
         $source_project = $source;
+        $additional_commands = [];
+        $no_install_flag = '';
```

Comment:
@kporras07 how about introducing  a unified variable for `create-project` command additional options, ie. `$create_project_options`?
```php
// ...
$source_project = $source;
$additional_commands = [];
$create_project_options = [];
// ...
if ('' !== $stability) {
    $create_project_options[] = "--stability $stability";
}
// ...
$create_project_options[] = '--no-install';
// ...
$create_project_command = sprintf(
    'composer create-project --working-dir=%s %s %s %s -n %s',
    $tmpsitedir, 
    $repository, 
    $source_project,
    $target,
    implode(' ', $create_project_options)
);
// ...
```

Reply:
Fixed. Thanks!